app/src/main/java/org/mozilla/focus/browser/integration/BrowserToolbarIntegration.kt

```diff
@@ -275,6 +277,23 @@ class BrowserToolbarIntegration(
         }
     }
 
+    @VisibleForTesting
```

Comment:
This annotation allows relaxing the visibility specifically for tests.
Will you add also add the tests for this method or can the annotation be removed and the method be marked as private?

Reply:
ok I will remove the annotation